.travis.yml

```diff
@@ -1,12 +1,15 @@
 language: go
 go:
-  - 1.7
-install: make install-ci
-env:
- # Set higher timeouts for Travis
- - TEST_TIMEOUT_SCALE=20 PACKAGE=github.com/m3db/m3db
+  - 1.8.3
```

Comment:
Yeah I was thinking of leaving 1.7 here too though, sorry should've been clear with that last comment.  Don't know what your thoughts are but we if we can support all historic Go versions we should try to.  So we should test with 1.7, 1.8 and when 1.9 is released with 1.9 too.